web/src/icons/code.tsx

```diff
@@ -9,6 +9,7 @@ const SvgCode = (props: SVGProps<SVGSVGElement>) => (
   >
     <path
       d="M10.6667 12L14.6667 8L10.6667 4M5.33334 4L1.33334 8L5.33334 12"
+      stroke="currentColor"
```

Comment:
Hmm, I believe this hardcodes the stroke-colour, making setting the stroke-colour from the calling context to be ignored (i.e., `<SvgCode className="... stroke-red-500" />` will not work).